eclipse-collections-forkjoin/pom.xml

```diff
@@ -63,7 +63,21 @@
         <plugins>
 
             <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>${jacoco.plugin.version}</version>
+            </plugin>
+
+            <plugin>
                 <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.19</version>
+                <configuration>
+                    <includes>
+                        <include>**/*Test.java</include>
+                    </includes>
+                    <argLine>-XX:-OmitStackTraceInFastThrow -Xms1024m -Xmx2048m @{argLine}</argLine>
+                    <runOrder>random</runOrder>
```

Comment:
@Godin should the argLine just be <argLine><argLine> or the whole argLine? If I understood correctly, JaCoCo will set the correct argLine for the relevant modules?